src/libYARP_OS/src/RosNameSpace.cpp

```diff
@@ -530,7 +530,7 @@ bool RosNameSpace::writeToNameServer(PortWriter& cmd,
     }
     bool ok = NetworkBase::write(getNameServerContact(),
                                  cmd2,
-                                 *(use_cache?&cache:&reply),
+                                 cache,
```

Comment:
I'm not sure about this, I want to check it a bit better to understand what was the original purpose of this.
Can you please remove this commit from the PR, so that we can merge this quickly and examine this commit later?